web/app/themes/clarity/error-pages/maintenance.html

```diff
@@ -70,7 +70,7 @@ <h1 class="o-title o-title--page">Website temporarily unavailable</h1>
     <div class="l-primary"></div>
     <div class="l-secondary c-copyright">
       <div class="c-copyright-notice">
-        <img src="/app/themes/clarity/dist/images/crown_copyright_logo.png" alt="crown copyright logo">
+        <img src="/app/themes/clarity/dist/images/tudor_crest.svg" alt="crown copyright logo">
```

Comment:
Have these logos been added in a previous commit? tudor_crest and moj_logo_header?

Reply:
Good spot, yes they were :)